Dockerfile

```diff
@@ -1,5 +1,5 @@
 # https://hub.docker.com/_/julia/
-ARG BASE_IMAGE=julia:1-buster
+ARG BASE_IMAGE=julia:1.6.2
```

Comment:
Avoids issues with automatically using the latest version of Julia 1+ which could break if the default memory usage increases.